ui/helpers/constants/common.ts

```diff
@@ -4,7 +4,8 @@ export const SECONDARY = 'SECONDARY';
 const _contractAddressLink =
   'https://support.metamask.io/managing-my-tokens/moving-your-tokens/why-am-i-being-warned-about-sending-tokens-to-a-contract/';
 
-export const TRANSACTION_SHIELD_LINK = 'https://metamask.io/transaction-shield';
+export const TRANSACTION_SHIELD_LINK =
```

Comment:
is this link used else where?

Reply:
This is incorrect. pls create a new page. 
Learn More in entry modal should go to old link (website)
here in claim view goes to support article